eve/utils.py

```diff
@@ -297,7 +297,11 @@ def filter_ignore_fields(d, fields):
         value_ = value
 
     h = hashlib.sha1()
-    h.update(dumps(value_, sort_keys=True).encode('utf-8'))
+    with app.app_context():
```

Comment:
you probably don't need tie app_context here since it's already there when this function is invoked from the test suite.